src/test/java/spoon/test/module/TestModule.java

```diff
@@ -313,9 +320,8 @@ public void testModuleNames() {
 		batchCompiler.configure(args);
 
 		CompilationUnit[] cu = batchCompiler.getCompilationUnits();
-		List list = new ArrayList<>(Arrays.asList(new String[]{String.copyValueOf(cu[0].module),String.copyValueOf(cu[1].module)}));
-		assertTrue(list.contains("foo"));
-		assertTrue(list.contains("bar"));
+		Set<String> list = Arrays.stream(cu).map(CompilationUnit::getModuleName).map(String::copyValueOf).collect(Collectors.toSet());
+		MatcherAssert.assertThat(list, is(Set.of("foo", "bar")));
```

Comment:
```suggestion
		assertThat(list, is(Set.of("foo", "bar")));
```

Reply:
My IDE had somehow lost track of the imports and marked everything as missing, so I didn't see the unused exports. Fixed after restart. I have cleaned it up and added a static import for the hamcrest assertThat as suggested by the junit deprecationcomment. Hope it's alright now.

Reply:
Yeah, I feel you. I find that most IDEs with default settings are over-eager when it comes to auto-importing stuff, it gets rather confusing rather quickly.